saleor/graphql/webhook/types.py

```diff
@@ -45,3 +109,7 @@ class Meta:
     @staticmethod
     def resolve_events(root: models.Webhook, *_args, **_kwargs):
         return root.events.all()
+
+    @staticmethod
+    def resolve_deliveries(root: models.Webhook, *_args, **_kwargs):
+        return core_models.EventDelivery.objects.filter(webhook_id=root.pk)
```

Comment:
Shouldn't we have here a dataloader?

Reply:
since it is FilterInputConnectionField with filtering and sorting we can't use data loaders at the moment.